orttraining/orttraining/training_api/checkpoint.h

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include "core/platform/path_lib.h"
+#include "core/platform/env.h"
+#include "onnx/defs/tensor_proto_util.h"
+#include "orttraining/core/framework/checkpointing.h"
+#include "orttraining/training_api/interfaces.h"
+
+namespace onnxruntime {
+namespace training {
+
+// TODO: Rename to api after all major classes implemented.
```

Comment:
Can we just rename this now? What is the advantage on holding on till later?

Reply:
Yeah, there is no big concerns, thought to avoid more changes. :) But looks no big deal, let me change it.

Reply:
Fixed